pkg/node/fsm/sync_state.go

```diff
@@ -272,9 +316,20 @@ func initSyncStateInFSM(state *StateData, fsm *stateless.StateMachine, info Base
 		proto.ContentIDMicroblock,
 		proto.ContentIDPBMicroBlock,
 		proto.ContentIDPBTransaction,
+		proto.ContentIDMicroBlockSnapshot,
+		proto.ContentIDMicroBlockSnapshotRequest,
+	}
+	if !info.enableLightMode {
+		syncSkipMessageList = append(syncSkipMessageList, []proto.PeerMessageID{
+			proto.ContentIDBlockSnapshot,
+		}...)
```

Comment:
```suggestion
		syncSkipMessageList = append(syncSkipMessageList, proto.ContentIDBlockSnapshot)
```
I think this looks simpler